src/Cabal2Nix/CorePackages.hs

```diff
@@ -8,7 +10,7 @@ module Cabal2Nix.CorePackages ( corePackages, coreBuildTools ) where
 -- The commented packages have previously been non-core, so we don't filter
 -- them.
 corePackages :: [String]
-corePackages = [
+corePackages = map toNixName [
```

Comment:
The lists are only used in one place, both before and after this patch. But sure, I'll move this closer to where it's used (when I'm back at my computer)